superset-frontend/src/components/Menu/Menu.jsx

```diff
@@ -59,6 +59,28 @@ const StyledHeader = styled.header`
     flex-direction: column;
     justify-content: center;
   }
+
+  .navbar-nav > li > a {
+    &:after {
+      content: '';
+      position: absolute;
+      bottom: -3px;
+      left: 0;
+      width: 100%;
+      height: 3px;
+      background-color: #20a7c9;
+      opacity: 0;
+      transition: opacity 0.5s;
```

Comment:
Similar to what @mistercrunch mentioned above, we also have a styling constant for animations; see here: https://github.com/apache-superset/superset-ui/blob/master/packages/superset-ui-style/src/index.ts#L125